scripts/build_utils.sh

```diff
@@ -1074,9 +1074,8 @@ get_nr_build_jobs() {
 }
 
 maybe_reset_ci_container() {
-    # ceph-container includes ceph-mgr-cephadm by default, but nautilus does
-    # not ship this package
-    if [[ "$CI_CONTAINER" && "$BRANCH" =~ nautilus ]]; then
+    # we don't build containers for nautilus or older
+    if [[ "$CI_CONTAINER" && "$BRANCH" =~ luminous|nautilus ]]; then
```

Comment:
i consider #1607 a workaround to stop the build failure before we have a decent fix addressing https://github.com/ceph/ceph-container/pull/1706#issuecomment-653034477,  because the container image build failure is now considered a fatal error.

if nobody is consuming the luminous and nautilus images, why are we building it in `build-push-ceph-container-imgs.sh`, just for the sake of completeness?

@leseb @dmick do you happen to know why?

Reply:
> if nobody is consuming the luminous and nautilus images, why are we building it in `build-push-ceph-container-imgs.sh`, just for the sake of completeness?

My guess is because when we started building containers, luminous and nautilus were the latest releases and CentOS8 wasn't out yet.

Reply:
I don't know who's consuming what Ceph release builds.  I do know the container build stuff seems to be getting more fragile as we add more special cases at various levels of the build.